resources/lib/translation.py

```diff
@@ -0,0 +1,81 @@
+import xbmcaddon
+from simple_logging import SimpleLogging
+
+log = SimpleLogging(__name__)
+addon = xbmcaddon.Addon()
+
+
+def i18n(string_id):
+    try:
+        return addon.getLocalizedString(STRINGS[string_id]).encode('utf-8', 'ignore')
+    except Exception as e:
+        log.warning('Failed String Lookup: %s (%s)' % (string_id, e))
```

Comment:
why not just info or even in this case since it is an error actually log it as an error, I dont see a time when you try to use a string and it is not available that it is not an error.